.travis.yml

```diff
@@ -4,8 +4,8 @@ os:
   - linux
   - osx
 julia:
-  - 0.5
-  - 0.6
+  - 1.0
+  - 1.1
```

Comment:
Although I would normally be concerned with backwards-compatibility for clients, the previous builds against v0.6 have been failing.